src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -390,6 +390,53 @@ You can create, read, update, or delete obfuscation expressions by using the New
   </Collapser>
 </CollapserGroup>
 
+## Sample expressions [#samples]
+
+Below, we've provided some sample regex expressions to obfuscate some of the most common sensitive data types. Note, Obfuscation Expressions must be created for each account where those expressions will be in use.
```

Comment:
No need to capitalize obfuscation expression here.

Reply:
got it.